lib/modules/activity/reply_card.dart

```diff
@@ -64,11 +64,12 @@ class ReplyCard extends StatelessWidget {
                       style: Theme.of(context).textTheme.labelSmall,
                     ),
                     const Spacer(),
-                    if (reply.user.id == Options().id) ...[
+                    ...[
                       Consumer(
                         builder: (context, ref, _) => SizedBox(
                           height: 40,
-                          child: Tooltip(
+                          child: reply.user.id == Options().id ?
+                          Tooltip(
```

Comment:
If you're using the dart formatter, can you make sure that this is its preferred indentation? I feel like the `Tooltip` would normally be indented one level further, but I may be off.

Reply:
I ran the Dart formatter, and it did... something